src/library_webgl.js

```diff
@@ -366,6 +366,9 @@ var LibraryGL = {
           if (GL.currentContext.version >= 2 && (dataType == 0x8368 /* GL_UNSIGNED_INT_2_10_10_10_REV */ || dataType == 0x8D9F /* GL_INT_2_10_10_10_REV */)) {
             sizeBytes = 4;
             break;
+          } else if (GL.currentContext.version >= 2 && dataType == 0x140B /* GL_HALF_FLOAT */) {
+            sizeBytes = 2;
+            break;
```

Comment:
Let's extract the `if (GL.currentContext.version >= 2)` check outside these two ifs to not be repeated?

Reply:
fixed